Formula/dnsdist.rb

```diff
@@ -0,0 +1,30 @@
+class Dnsdist < Formula
+  desc "Highly DNS-, DoS- and abuse-aware loadbalancer"
+  homepage "https://www.dnsdist.org/"
+  url "https://downloads.powerdns.com/releases/dnsdist-1.2.0.tar.bz2"
+  sha256 "9885c9ee8ac7076aede586ea58d4642eb877e7b2d76c902254494e2a5a5faa78"
+
+  depends_on "pkg-config" => :build
+  depends_on "lua"
+  depends_on "libedit"
```

Comment:
macOS comes with libedit

Reply:
do you have hints on how to use it with autotooled projects? My personal mac has a .pc file but I understand others do not and autoconf is not picking it up on Jenkins.

Reply:
```
iMac-TMP:~ joe$ brew linkage dnsdist
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
  /usr/lib/libedit.3.dylib
  /usr/lib/libncurses.5.4.dylib
Homebrew libraries:
  /usr/local/opt/lua/lib/liblua.5.2.dylib (lua)
Dependencies with no linkage:
  boost
iMac-TMP:~ joe$ 
```
seems to work no?

Is boost build time only?

Reply:
this dep fails only on High Sierra on your Jenkins

Reply:
lol looks like apple removed the .pc for libedit on 10.13 … joy

Reply:
something like
```
    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
    ENV["LIBEDIT_CFLAGS"] = "-I#{sdk}/usr/include -I#{sdk}/usr/include/editline"
    ENV["LIBEDIT_LIBS"] = "-L/usr/lib -ledit -lcurses"
```

Reply:
> Is boost build time only?

yes, will fix